### ISSUE TITLE
steamdeck-kde-presets fails to install

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#583]
```
error: failed to commit transaction (conflicting files)
steamdeck-kde-presets: /etc/xdg/konsolerc exists in filesystem (owned by konsole)
```

We may want to provide our own package specifically for the theme in the AUR. This way, we do not install all of the extra things included as part of the `steamdeck-kde-presets` package.

**Comment by LukeShortCloud** [comment]
/etc/xdg/konsolerc variants:

- `steamdeck-kde-presets`
    ```
    [Desktop Entry]
    DefaultProfile=Vapor.profile
    ```
- `konsole`
    ```
    [MainWindow]
    MenuBar=Disabled
    ```

Once we get a separate package for the Vapor theme, we can easily configure this with `crudini`.

Related: https://github.com/LukeShortCloud/winesapOS/issues/590

**Comment by mark-monteiro** [comment]
I was hoping to upgrade my system and ran into this issue (I also ran into #593, but that was fairly straightforward to fix). I'm not very familiar with Arch or the sub-systems affected here, but from my interpretation of the referenced commit with the fix for this issue, it looks like I could run the following commands to get that fix applied to my system:

```sh
crudini --set /etc/xdg/konsolerc "Desktop Entry" DefaultProfile Vapor.profile
# Remove the whitespace from the lines that 'crudini' creates.
sed -i -r "s/(\S*)\s*=\s*(.*)/\1=\2/g" ${WINESAPOS_INSTALL_DIR}/etc/xdg/konsolerc

#  Replace steamdeck-kde-presets with vapor-steamos-theme-kde
sudo pacman -R -n --noconfirm steamdeck-kde-presets
yay_install_chroot vapor-steamos-theme-kde
```

Does that look right? Is there anything else I would be missing? For context, I'm currently running release 3.2.1 of winesapOS, I just installed it a few days ago.

I guess more importantly, is it a good idea in general to apply the fix manually? Or should I just waiting for the next winesapOS release? I'm not sure how the winesapOS update process works so I wasn't sure if running commands like this manually would conflict.

**Comment by LukeShortCloud** [comment]
@mark-monteiro Yes, that should be safe to run.

I just released winesapOS 3.3.0 which addresses your upgrade concerns. You can now run the "winesapOS Upgrade" desktop shortcut and it will make that change along with many other improvements.

https://github.com/LukeShortCloud/winesapOS/releases/tag/3.3.0

**Comment by mark-monteiro** [comment]
🚀 Perfect! Thanks for following up
